diff --git a/lib/index.js b/lib/index.js
index 1ca507a9bc2bcddbdc4eedf097075e538fc7ce31..d47d5d9621e2fbc61ee9a103e9487af84b8d5af7 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -18,7 +18,8 @@ function normalizeOptions(options = {}) {
   let {
     allowNamespaces = true,
     jsxPragma,
-    onlyRemoveTypeImports
+    onlyRemoveTypeImports,
+    yeetEmptyImportElisions
   } = options;
   const TopLevelOptions = {
     ignoreExtensions: "ignoreExtensions",
@@ -27,6 +28,7 @@ function normalizeOptions(options = {}) {
     jsxPragma: "jsxPragma",
     jsxPragmaFrag: "jsxPragmaFrag",
     onlyRemoveTypeImports: "onlyRemoveTypeImports",
+    yeetEmptyImportElisions: "yeetEmptyImportElisions",
     optimizeConstEnums: "optimizeConstEnums",
     rewriteImportExtensions: "rewriteImportExtensions",
     allExtensions: "allExtensions",
@@ -56,6 +58,7 @@ function normalizeOptions(options = {}) {
     jsxPragma,
     jsxPragmaFrag,
     onlyRemoveTypeImports,
+    yeetEmptyImportElisions,
     optimizeConstEnums,
     rewriteImportExtensions
   };
@@ -119,6 +122,7 @@ var index = helperPluginUtils.declarePreset((api, opts) => {
     jsxPragma,
     jsxPragmaFrag,
     onlyRemoveTypeImports,
+    yeetEmptyImportElisions,
     optimizeConstEnums,
     rewriteImportExtensions
   } = normalizeOptions(opts);
@@ -129,6 +133,7 @@ var index = helperPluginUtils.declarePreset((api, opts) => {
     jsxPragma,
     jsxPragmaFrag,
     onlyRemoveTypeImports,
+    yeetEmptyImportElisions,
     optimizeConstEnums
   });
   const getPlugins = (isTSX, disallowAmbiguousJSXLike) => {
